lib/hsa/hsaBeamformTranspose.cpp

```diff
@@ -55,9 +57,37 @@ hsa_signal_t hsaBeamformTranspose::execute(int gpu_frame_id, hsa_signal_t preced
     params.num_dims = 2;
 
     params.private_segment_size = 0;
-    params.group_segment_size = 8192;
+    params.group_segment_size = 33*32*4;
```

Comment:
This could use a comment to explain where it's coming from, or turn it into self-descriptive constants.